README.md

```diff
@@ -115,7 +118,8 @@ selfClosing: [ 'img', 'br', 'hr', 'area', 'base', 'basefont', 'input', 'link', '
 // URL schemes we permit
 allowedSchemes: [ 'http', 'https', 'ftp', 'mailto' ],
 allowedSchemesByTag: {},
-allowProtocolRelative: true
+allowProtocolRelative: true,
+allowedIframeDomains: ['youtube.com', 'vimeo.com']
```

Comment:
This sometimes-incorrect example, and the other example below where you attempt to document the need to "also include a hostname," plus the fact that the url parser returns a property called "hostname" and that's what you are actually comparing to, together convince me that this option really needs to be called `allowedIframeHostnames`. Then it is clear what you must do (list every allowed hostname, not just the domain name, or it won't be accepted).

Reply:
Agreed!